internal/handler/wallet/delete.go

```diff
@@ -21,10 +21,10 @@ type walletDelete struct {
 }
 
 // HandleDelete handles a delete request
-func (w *walletDelete) Handle(params operations.RestWalletDeleteParams) middleware.Responder {
+func (w *walletDelete) Handle(params operations.RestAccountDeleteParams) middleware.Responder {
 	wallet, err := wallet.Load(params.Nickname)
 	if err != nil {
-		return operations.NewRestWalletDeleteBadRequest().WithPayload(
+		return operations.NewRestAccountDeleteBadRequest().WithPayload(
```

Comment:
I would add:
- 401 for password authentification refused (cancelled) or timeout (408 is overkill i think)
- 403 for bad password
All the rest are ok

Reply:
ok

FYI: there is no way for bad password because we keep the prompt open until either the password is right or the user closes the prompt or clicks on cancel button.